pages/rfc/distributed_registry.rst

```diff
@@ -0,0 +1,224 @@
+
+Limitations:
+
+* For now, we only support 1 node per vendor.
+* For now, we only support 1 vendor per care organisation (but a vendor can serve multiple care organisations).
+
+Steps:
+
+#. Node registration:
+
+    #. The vendor generates a CSR (for the node's CA certificate) and submits it to Nuts Discovery.
+    #. A Nuts administrator accepts the CSR and issues the node CA certificate.
+    #. The vendor issues node identity certificate (for its node) using its CA.
+    #. The vendor submits its node information (node identity certificate, node address, ...) to Nuts Discovery.
+    #. The node information is published so that every (other) node in the network knows about the vendor's node.
+
+#. Vendor registration:
+
+    #. The vendor generates a CSR (for the vendor's CA certificate) and submits it to Nuts Registry.
+    #. A Nuts administrator accepts the CSR and issues the vendor CA certificate.
+    #. The vendor issues its vendor certificate using its CA certificate. This certificate is used to sign events.
+    #. The vendor creates the registration event and signs it using the vendor certificate.
+
+Vendor CA certificate fields:
+
+====================  =====
+Field                 Value
+====================  =====
+Validity              3 years
+Key usage             Digital signature
+CA?                   true
+Issuer                Nuts Vendor CA
+Subject.CN            Vendor's name or unit within the organisation
+Subject.O             Vendor's name
+Subject.ST            Province in which the vendor is located
+Subject.C             Vendor's country
+Subject.serialNumber  Chamber of Commerce registration number of the vendor
+====================  =====
+
+Vendor Registration Event Payload
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+======================  =====
+Field                   Format
+======================  =====
+Vendor CA certificate   JWK encoded X.509
+======================  =====
+
+.. note::
+  We could add the vendor CA certificate to the X.509 trust chain when a certificate is referenced,
+  making this event unnecessary. However, this bloats the event header so it's desirable to register it once using
+  this event so it can be looked up every time a certificate (issued by this CA certificate) is validated.
+
+Registering a vendor - care organisation relation
+-------------------------------------------------
+
+Through this process a vendor registers a care organisation as its client.
+Afterwards, the vendor can register endpoints for the care organisation it serves through its node.
+
+Steps:
+
+#. The vendor issues a certificate for the care organisation under its vendor CA certificate
+   (see the table below for the prescribed fields of the certificate).
+#. The vendor creates the claim event and signs it using the (just issued) .
+
+Care organisation certificate fields:
+
+====================  =====
+Field                 Value
+====================  =====
+Validity              3 years
```

Comment:
age should be set to 1 year I think untill we get some comments/a good reason to do something else.